### PR TITLE
Missing raml fix [WIP]

### PR DESCRIPTION
**Pull request by davifantasia**
This fixes missing api specs in the UI:

https://commercetools-docs-git-renovate-all.commercetools.now.sh/import-api/import-operation

### CODE REVIEW

**Comment by emmenko** [comment]
Thanks, so we can then release this and things should work again?

**Comment by emmenko** [comment]
Don't forget to assign the labels to PRs

**Comment by davifantasia** [comment]
> Thanks, so we can then release this and things should work again?

It should work but still needs to be tested as I can't yet find a way to test locally until it's available on NPM.

File: websites/import-api/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Import API
```
Comment (emmenko) [inline websites/import-api/README.md]: Don't commit this here please!

**Pull request closed without merging**